security/remote_devices/spiffe_and_ssh/remote/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ networks:
     driver: bridge
 services:
   sshd-remote:
-    image: edgex-sshd-remote:latest
+    image: edgex-sshd-remote:3.0.0
```

Comment:
This won't work.  This is a locally built image.

Reply:
in the instruction it uses `docker compose build` so it can build the locally image with 3.0.0

Reply:
i can build it without any issue

Reply:
true, though it is not at all tied to edgex release version.

Reply:
i'm fine either way, it just that this update tied to version 3.0.0 in some extent

Reply:
What is building this image? `docker-compose build` ?? If so, nothing is tagging it as 3.0.0??

All referenced to `docker-compose` should be changed to `docker compose`

Reply:
Oh, I think the build will tag it since it doesn't find it, correct?

Reply:
updated